core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -23,6 +23,10 @@
 
 public final class CarbonCommonConstants {
   /**
+   * Boolean size in bytes
+   */
+  public static final int BOOLEAN_SIZE_IN_BYTE = 1;
```

Comment:
This is not required, it should be added in DataType enum

Reply:
ok